bool gps_status
bool imu_status
bool dgps_status
uint8 no_of_satellites
uint32 min_since_1980
